spec/features/staff/users_can_choose_recipient_country_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     context "with JavaScript disabled" do
       scenario "countries are choosen from a select box" do
-        expect(page).to have_select(I18n.t("page_title.activity_form.show.country"))
```

Comment:
I wasn't expecting page_title to be affected by any changes we made to the form builder gem. The `@page_title` is set in the `ActivityFormController` and then passed to the `_wrapper` on each form step. This allows us to keep our form titles different from individual field names.

Having a quick look I think we've accidentally broken page titles for country prior to this work?

Our en.yml says:

```
  page_title:
    activity_form:
      show:
        country: What country will benefit from this activity?
```

But the form view on staging just has "country" and I think we want the former which would make these switches to activerecord.etc no longer needed!
![Screenshot 2020-05-29 at 10 58 06](https://user-images.githubusercontent.com/912473/83248443-f96f1c00-a19c-11ea-95c4-03d748f7e1f7.png)

Page titles seem to be working for other fields like identifier and sector.